README.md

```diff
@@ -143,7 +141,7 @@ Furthermore, the project OpenRobertaServer contains in directory staticResources
 * HTML and CSS
 * Javascript libraries based on jquery and bootstrap for the frontend
   * assertions (DBC), ajax-based server calls (COMM), logging (LOG) and
-  * javascript resources for blockly (see: http://code.google.com/p/blockly/)
+  * javascript resources for [blockly](http://code.google.com/p/blockly/)
```

Comment:
Please use the updated link from the other merged PR here.

Reply:
Made the changes.